### ISSUE TITLE
Apparently utested prototype code breaks the appeals process.

### ISSUE THREAD

**Issue by GWHAYWOOD** [UTRS2/utrs#461]
Please see

https://en.wikipedia.org/w/index.php?title=User_talk:Ged_Haywood

and

https://utrs-beta.wmflabs.org/public/appeal/view?hash=d9e925a18958674c9dcdbb52e57396f3

In which I asked for the IP address range 2001:0470:6976::0/48 to be unblocked.

The API for the appeal process is broken.

I made the appeal from my account, and the appeal page initially listed the appeal in my name - that is, the (left hand) column headed "User" contained my account name.  Note that there is another column in the same table, also headed "User", which bodes ill.

After the page was updated with a comment from an administrator, my name in the left-hand column was replaced by the IPv6 CIDR (2001:470:4000::0/34) in which my /48 resides.

After this bowdlerization of the appeal, the reply was that I should use my account, not the IP address!

That is in fact what I did.

The impression given is not, I think, one of which anyone at Wikimedia would be proud.

**Comment by supertassu** [comment]
Closing as a duplicate of #458.
